base/src/main/java/io/spine/validate/MessageValidator.java

```diff
@@ -254,10 +257,8 @@ public void visitCustom(CustomConstraint constraint) {
 
     private static void checkTypeConsistency(Range<ComparableNumber> range, FieldValue value) {
         if (range.hasLowerBound() && range.hasUpperBound()) {
-            NumberText upper = range.upperEndpoint()
-                                    .toText();
-            NumberText lower = range.lowerEndpoint()
-                                    .toText();
+            NumberText upper = range.upperEndpoint().toText();
+            NumberText lower = range.lowerEndpoint().toText();
             if (!upper.isOfSameType(lower)) {
                 String errorMessage = "Boundaries have inconsistent types: lower %s, upper %s";
```

Comment:
Please backtick type names and finish the message with period. Also, please have “is” after “lower” and “upper”.